### ISSUE TITLE
#-943847739------------- Links must have discernible text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#491]
<strong>Source:</strong>`<a id="edit-button-434c061b20f74696a4619513b2811ca1" href="/layers/434c061b20f74696a4619513b2811ca1/edit" aria-describedby="calcite-tooltip-080de8b2-090d-9dd0-a624-d709ad1052f0">`
<strong>Element Location: </strong>#edit-button-434c061b20f74696a4619513b2811ca1
<strong>Impact: </strong>serious
<strong>failureSummary:</strong> Fix all of the following:
  Element is in tab order and does not have accessible text

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attributeTags: [Ljava.lang.String;@5487a8f1
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
